kolibri/auth/migrations/0003_auto_20160808_1845.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.7 on 2016-08-08 18:45
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('kolibriauth', '0002_auto_20160318_0557'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='role',
+            name='user',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='roles', to='kolibriauth.FacilityUser'),
+        ),
+    ]
```

Comment:
@DXCanas Please delete this file before merging; re-generating all migrations on master